DTO/CategoryStoreDTO.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\DTO;
+
+class CategoryStoreDTO
+{
+    public string $name;
+
+    public function __construct(string $name)
+    {
+        $this->name = $name;
+    }
+}
```

Comment:
```suggestion
class CategoryStoreDTO
{
    public function __construct(public string $name)
    {
    }
}
```

попробуй после этого более краткий способ